eng/common/scripts/Verify-Links.ps1

```diff
@@ -269,23 +284,38 @@ while ($pageUrisToCheck.Count -ne 0)
 
   $linkUris = GetLinks $pageUri
   Write-Host "Found $($linkUris.Count) links on page $pageUri";
-  
+  $badLinksPerPage = @();
   foreach ($linkUri in $linkUris) {
+    Write-Host "before the replace $linkUri"
+    Write-Host "the commit $branchReplacementName"
     $linkUri = ReplaceGithubLink $linkUri
-
+    Write-Host "after the replace $linkUri"
```

Comment:
We should remove all these extra Write-Host once you are doing testing as there is already enough noisy output in this script. You can consider leaving some as Write-Verbose if they are interesting.

Reply:
This is for testing only.

Reply:
Will remove all

Reply:
Ok great lets remove these 3 newly added Write-Host logging steps here once you finish your testing and before merging.